Sub/Src/ComputerVision/Nodes/CamNode.py

```diff
@@ -0,0 +1,131 @@
+import numpy as np
+import cv2
+import math
+import random
+import socket
+import sys
+import struct
+import math
+import io
+import time
+from ctypes import *
+from libs.darknet import *
+from MechOS.message_passing.Nodes.node_base import node_base
+
+class CamNode(node_base):
+
+    def __init__(self, MEM, IP):
+
+        # IP and MEM RAM locations
+        node_base.__init__(self, MEM, IP)
+
+        # Instantiations
+
+        #--MESSAGING INFO--#
+        self.MAX_UDP_PACKET_SIZE = 1500
+
+        # The end byte of the image sent over udp
+        self.END_BYTE = bytes.fromhex('c0c0')*2
+
+        # the Max Packet Size the Port will accept
+        self.MAX_PACKET_SIZE = 1500
+
+        #--CAMERA INSTANCE--#
+        self.capture = cv2.VideoCapture(1)
+
+        # Maximum image size the Tegra allows without timeout
+        self.capture.set(3, 450)
+        self.capture.set(4, 450)
+
+        # Neural Network Loaded from instance in darknet module
+        self.net  = load_net(b'./libs/darknet/cfg/yolov3-tiny.cfg',
```

Comment:
I highly recommend having this configurable and the ability to reload different weights without restarting the program. That was one thing I wanted to have the ability to do when I was testing and I just never had the time to do it.

Reply:
@PierceATronics Parameter Server Integration, can you take care of this?

Reply:
Yes I will integrate this